lib/jekyll-redirect-from/redirector.rb

```diff
@@ -4,66 +4,66 @@ class Redirector < Jekyll::Generator
 
     def generate(site)
       original_pages = site.pages.dup
-      generate_alt_urls(site, site.posts) if Jekyll::VERSION < '3.0.0'
+      generate_alt_urls(site, site.posts) if Jekyll::VERSION < "3.0.0"
       generate_alt_urls(site, original_pages)
       generate_alt_urls(site, site.docs_to_write)
     end
 
     def generate_alt_urls(site, list)
       list.each do |item|
-        if has_alt_urls?(item)
+        if alt_urls?(item)
```

Comment:
Technically, technically, I think we'll want to alias the old versions of these to avoid a breaking change, since these were public methods (although I don't know how/why you'd use them).